requirements.txt

```diff
@@ -1,8 +1,8 @@
 #
-# This file is autogenerated by pip-compile
-# To update, run:
+# This file is autogenerated by pip-compile with Python 3.8
+# by the following command:
 #
-#    pip-compile --output-file=requirements.txt requirements.in setup.py
```

Comment:
We don't use `--annotation-style elsewhere` in the other REANA components, so it's best to leave it out for now.